dashboard/app/bisect_test.go

```diff
@@ -188,11 +188,13 @@ For information about bisection process see: https://goo.gl/tpsmEJ#bisection
 
 		c.expectEQ(msg.To, []string{
 			"author@kernel.org",
-			"bugs@syzkaller.com",
-			"default@maintainers.com",
 			"reviewer1@kernel.org",
```

Comment:
If there is only one commit, the procedure was to merge `Commit.CC` to `BugReport.Maintainers`, which explains the commit reviewers here. This should change for new users when we make changes on `pkg/vcs/git.go` to divide commitCC into To/Cc (they are currently being assigned to To).

Reply:
Okay, so this is testing the case where dashapi.Commit uses the old `Cc` field instead of the new `Recipients` field.  This wasn't obvious to me.  Is there a way that you could refactor the test to test both the old and new fields?